<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extraction Piste Son V0.3</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="dockWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>215</width>
      <height>547</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>215</width>
      <height>542</height>
     </size>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="QLabel" name="labelPanneau">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Panneau de controle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>42</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_Version">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>507</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Ver: v1.0  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="BoutonOuvrir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>189</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ouvrir Fichier MIDI</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitre">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>39</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Commande</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonPause">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPlay">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSpeed">
         <property name="suffix">
          <string>x</string>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>196</width>
        <height>17</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="sliderTimeline">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTime">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTotal">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>382</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>382</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitre_2">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>380</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Lecteur MIDI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EnregistrerPDF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>189</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enregistrer PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Imprimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>189</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>766</width>
      <height>529</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Technicien1">
     <attribute name="title">
      <string>SÃ©paration Piste</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>711</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Tab_Wybauw1">
          <attribute name="title">
           <string>SÃ©paration</string>
          </attribute>
          <widget class="QPushButton" name="BoutonOuvrirWybauw">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>70</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Ouvrir un fichier audio</string>
           </property>
          </widget>
          <widget class="QPushButton" name="IsolerFreqBasseWybauw">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>240</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Isoler frÃ©quences basses</string>
           </property>
          </widget>
          <widget class="QPushButton" name="IsolerFreqHauteWybauw">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>240</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Isoler frÃ©quences hautes</string>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonQuitWybauw">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>320</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Quitter</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="SpinBoxWybauw">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>160</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>500000</number>
           </property>
          </widget>
          <widget class="QLineEdit" name="TextWybauw">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>160</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>FrÃ©quence d'Ã©chantillonnage</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Technicien2">
     <attribute name="title">
      <string>CrÃ©ation</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>711</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="AffichagePartition">
          <attribute name="title">
           <string>Affichage Partition</string>
          </attribute>
          <widget class="QPushButton" name="BoutonConversion">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>90</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CrÃ©ation Partition</string>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonPartition">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>200</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ouvrir Partition</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelPrevisualisation">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>40</y>
             <width>261</width>
             <height>341</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>72</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonPrevisualisation">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>40</y>
             <width>261</width>
             <height>341</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonModifier">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>310</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Modifier Partition</string>
           </property>
          </widget>
          <widget class="Line" name="line_3">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>20</y>
             <width>85</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="labelTitrePrevisualisation">
           <property name="geometry">
            <rect>
             <x>482</x>
             <y>15</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>PrÃ©visualisation</string>
           </property>
          </widget>
          <widget class="Line" name="line_4">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>20</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="ModeDeveloppeur">
          <attribute name="title">
           <string>Piano Roll</string>
          </attribute>
          <widget class="QLabel" name="labelEdition">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>681</width>
             <height>411</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Technicien3">
     <attribute name="title">
      <string>Modification</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>721</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Tab_Milon1">
          <attribute name="title">
           <string>Modifier</string>
          </attribute>
          <widget class="QPushButton" name="BoutonOuvrirMilon">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>50</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Ouvrir Fichier</string>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonSaveMilon">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>50</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Sauvegarder</string>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonPlayMilon">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/Icon/play.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonPauseMilon">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/Icon/pause.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="BoutonModifierMilon">
           <property name="geometry">
            <rect>
             <x>245</x>
             <y>260</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Modifier la note</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelCle">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>120</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="labelNote">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>120</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="labelBarre">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>120</y>
             <width>621</width>
             <height>101</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/Icon/BarreMilon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QSlider" name="horizontalSlider">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>20</y>
             <width>561</width>
             <height>20</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelCheminAcces">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>535</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Acces">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>537</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chemin d'accÃ¨s :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>BoutonQuitWybauw</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
